codersdk/deployment.go

```diff
@@ -997,6 +1014,135 @@ when required by your organization's security policy.`,
 		Group:         &deploymentGroupIntrospectionLogging,
 		YAML:          "filter",
 	}
+	emailFrom := serpent.Option{
+		Name:        "Email: From Address",
+		Description: "The sender's address to use.",
+		Flag:        "email-from",
+		Env:         "CODER_EMAIL_FROM",
+		Value:       &c.Notifications.SMTP.From,
```

Comment:
Why are we keeping `.Notifications` here? Rename to email?

Reply:
I kept `.Notifications` for now because separating out `SMTP` from it would be a larger refactor and at the moment we only use `SMTP` for notifications so didn't think it warranted being put into this PR.